TOAST/application/controllers/test/User_test.php

```diff
@@ -12,15 +12,15 @@ function __construct()
 		parent::__construct(__FILE__);
 		$this->UserController = new User(true);	//Instantiate class
 	}
-
+/*
```

Comment:
I commented out the insert (and all the pre) because I don't have a real user class !